agimus_controller/agimus_controller/factory/robot_model.py

```diff
@@ -58,12 +60,26 @@ def __post_init__(self):
                 f"Armature must have the same shape as q0. Got {self.armature.shape} and {self.q0.shape}."
             )
 
-        # Ensure paths are valid strings
-        if not self.urdf_path.is_file():
-            raise ValueError("urdf_path must be a valid file path.")
+        # Ensure URDF and SRDF are valid
+        if self.urdf is not None:
+            if isinstance(self.urdf, Path) and not self.urdf.is_file():
+                raise ValueError(
+                    "urdf must be a valid file path. "
+                    f"File: '{self.urdf}' doesn't exist!"
+                )
+            elif self.urdf == "":
+                raise ValueError("urdf can not be an empty string.")
+
```

Comment:
I forgot how to make multiple lien suggestion, but this should be simplified since no None posib;e